config/Version.public.xcconfig

```diff
@@ -1,4 +1,4 @@
-VERSION_SHORT=20.6
+VERSION_SHORT=20.7
 
 // Public long version example: VERSION_LONG=9.9.0.0
-VERSION_LONG=20.6.0.3
+VERSION_LONG=20.7.0.0
```

Comment:
While I run multiple release build jobs in CI, only the last one succeeded, so there was no need to further bump the build number.